app/components/metrics_summary_component/view.rb

```diff
@@ -12,7 +12,7 @@ def initialize(form_live_date, metrics_data)
 
       if metrics_data.nil?
         @error_message = I18n.t("metrics_summary.errors.error_loading_data_html")
-      elsif metrics_data[:form_is_new]
+      elsif @start_date == Time.zone.today
```

Comment:
Good shout - have updated! Added one for `form_went_live_yesterday?` as well, even though we only check this once, since it's nice and clear and keeps things consistent with the new `form_went_live_today?` method

Reply:
Thats a lot easier to read 👍